MoonDawgArduino/MoonDawgArduino/MoonDawgArduino.ino

```diff
@@ -133,5 +157,6 @@ void loop() {
 
 
     }
+    //camera.hold();
```

Comment:
what is this for

Reply:
Another relic of the hold function. Will be removed.